Library/Homebrew/sandbox.rb

```diff
@@ -141,8 +141,15 @@ def path_filter(path, type)
   class SandboxProfile
     SEATBELT_ERB = <<-EOS.undent
       (version 1)
+      (deny network-inbound)
+      (deny network-outbound)
       (debug deny) ; log all denied operations to /var/log/system.log
       <%= rules.join("\n") %>
+      (allow network-outbound
+        (subpath "/tmp")
+        (subpath "/private")
+        (subpath "#{HOMEBREW_TEMP}")
```

Comment:
Have you confirmed that the path is used for downloading files? Given my previous experience these paths should be applied to the executable files, like `curl` etc.